src/vite-types.d.ts

```diff
@@ -22,9 +22,7 @@ interface DbRepo {
   readonly language?: string;
   readonly license?: string;
   readonly url?: string;
-  readonly contributions: DbContribution[];
-  readonly starsRelation: { starsCount: number }[],
-  readonly votesRelation: { votesCount: number }[],
+  readonly contributionsCount?: number;
```

Comment:
this type here, mirroring the backend types, should not be removed just because we commented out the avatars; could maybe figure out a way to manage updates to types in a more streamlined fashion.

Reply:
linking for to reference there #333, but perhaps I create another PR to beta adding these back in.

Reply:
let's resolve the ones you created an issue for 🍕